### ISSUE TITLE
Adaptive LayerNorm may cause loss explosion in QwenPI

### ISSUE THREAD

**Issue by Yysrc** [starVLA/starVLA#237]
I fine-tuned QwenPI on the 27,500 RoboTwin datasets, and the loss exploded around step 50,000 during training (Figure 1). Notably, this issue did not occur when fine-tuning a single subtask of RoboTwin or when fine-tuning on the LIBERO dataset. 

Changing `ada_norm` to `norm` in the config file and replacing the adaptive layer norm in the last layer of `_process_output` with a standard LayerNorm resolved the problem (Figure 2). 

In addition, I observed that QwenPI already incorporates timestep embeddings when inputting actions, while also injecting timestep information via adaptive layer norm. In typical DiT architectures, only one of these two methods is used, which may result in redundancy.

<img width="967" height="460" alt="Image" src="https://github.com/user-attachments/assets/93ce5a0f-d674-4a93-8e14-b845d29730da" />

<img width="977" height="574" alt="Image" src="https://github.com/user-attachments/assets/d0d96f13-6654-431a-9f25-4f98d8c1b0d4" />
